html-aam/index.html

```diff
@@ -6860,19 +6869,30 @@ <h4 id="el-table">`table`</h4>
               <th>
                 <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
               </th>
-              <td>Use WAI-ARIA mapping</td>
+              <td>
+                <div class="role">Use WAI-ARIA mapping</div>
+                <div class="relations"><span class="type">Relations:</span> `IA2_RELATION_LABELLED_BY` with first instance of child <a href="#el-caption">`caption`</a> element</div>
+              </td>
             </tr>
             <tr>
               <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
-              <td>Use WAI-ARIA mapping</td>
+              <td>
+                <div class="role">Use WAI-ARIA mapping</div>
+              </td>
             </tr>
             <tr>
               <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
-              <td>Use WAI-ARIA mapping</td>
+              <td>
+                <div class="role">Use WAI-ARIA mapping</div>
+                <div class="relations"><span class="type">Relations:</span> `ATK_RELATION_LABELLED_BY` with first instance of child <a href="#el-caption">`caption`</a> element</div>
+              </td>
             </tr>
             <tr>
               <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
-              <td>Use WAI-ARIA mapping</td>
+              <td>
+                <div class="role">Use WAI-ARIA mapping</div>
+                <div class="role-namefrom"><strong>AXDescription:</strong> value from child <a href="#el-caption">`caption`</a> subtree</div>
```

Comment:
Subtree seems to imply DOM subtree such as `element.innerText`, but I think what you want here is the computed name of the first rendered caption element that is a child(?) of the table. I did not provide an inline diff because I think some of these suggestions also apply to the above changes.. For example, the "hidden" stipulation is presumably a normative requirement, that is listed in an informative note.... I think adding "rendered" (as I did here) in all normative contexts may resolve that, but there are other ways to make the change too.

Reply:
thanks for the catch on the normative requirement in a note.  i'll work on that asap.

i did however intend the subtree to be where the name was derived from, rather than just stating name in general, since authors shouldn't be using things like aria-label to name a legend or caption. 

html aam uses 'subtree' in a lot of the name calculation steps that were written far before i started editing the spec.  so if we want to change the way that's referenced, i think we need to do that in many places (though we can start with this PR)